NET_project/Benchmarks/NBody.cs

```diff
@@ -31,9 +31,8 @@ public unsafe struct NBodyNET : IJob {
 	}
 
 	[MethodImpl(MethodImplOptions.AggressiveInlining)]
-	private void InitializeBodies(NBody* sun, NBody* end) {
-		const double pi = 3.141592653589793;
-		const double solarMass = 4 * pi * pi;
+	private static void InitializeBodies(NBody* sun, NBody* end) {
```

Comment:
NBody in particular could get a huge speedup if it used `Vector128<double>` and/or some unsafe tricks so that the planetary body information was an RVA static that we loaded from instead.

It's the type of thing where a hotpath would want to do such a trick and define a helper to make it safe/simpler for regular code to access, abstracting away the unsafe.